README.md

```diff
@@ -1,18 +1,18 @@
 # RoboRacing Software [![CircleCI](https://circleci.com/gh/RoboJackets/roboracing-software.svg?style=svg)](https://circleci.com/gh/RoboJackets/roboracing-software)
 
-<img src="https://raw.githubusercontent.com/wiki/RoboJackets/roboracing-software/images/buzz_bigoli.jpg" style="max-height=400px;">
+<img src="https://raw.githubusercontent.com/wiki/RoboJackets/roboracing-software/images/sedanii.png" width="1500" height="500">
 
-This repository contains [ROS](http://ros.org) packages for the [RoboJackets](http://robojackets.org) RoboRacing team.
+Welcome to the [RoboJackets](http://robojackets.org) RoboRacing software repository! This document will give you a brief outline of the repository's layout and some simple instructions for setting up the project. For more detailed information, please visit the [wiki](https://github.com/RoboJackets/roboracing-software/wiki).
```

Comment:
I would either link to the robojackets wiki or update the github wiki. Currently neither has all the information, so I would combine it and put it on wiki.robojackets.org .

Reply:
I just linked wiki.robojackets.org